src/nfa/operations.cc

```diff
@@ -15,6 +15,9 @@
  * GNU General Public License for more details.
  */
 
+//is it good that header file names don't match cc files? What is the system actually? Is there a difference between operations and algorithms?
+//Generally, a comment on what the file is for would be good everywhere.
```

Comment:
This will be added when I get to a refactorization of  headers of source and header files.

Reply:
ok